items.md

```diff
@@ -283,6 +283,51 @@ the discoverable operation will result in:
 ```
 ## Linked entities
 
+### Access Status
+
+**GET /api/core/items/<:uuid>/accessStatus**
+
+This endpoint expose the mechanism for retrieving and calculating the access status of a DSpace item.
+It can be checked by calling this endpoint with the the corresponding item UUID.
+
+```
+curl -v "http://{dspace-server.url}/api/core/items/2245f2c5-1bed-414b-a313-3fd2d2ec89d6/accessStatus"
+```
+
+This will return the access status, E.G.:
+
+_200 - Response if the UUID parameter is valid_
+```json
+{
+  "status": "metadata.only",
+  "type": "accessStatus",
+  "_links" : {
+    "self" : {
+      "href" : "http://{dspace-server.url}/api/core/items/2245f2c5-1bed-414b-a313-3fd2d2ec89d6/accessStatus"
+    }
+  }
+}
+```
+
+Fields
+- Status: String value if the UUID is valid
+- Type: Type of the endpoint, "accessStatus" in this case
+
+Exposed links:
+- self: The valid URL to the item's access status
+
+Default access status values
+- metadata.only
+- open.access
+- embargo
+- restricted
+- unknown
```

Comment:
We might want a basic description after each of these default status values, e.g.:
```
- metadata.only = Item only includes metadata (no files)
- open.access = Item's primary file is downloadable to anonymous users
- embargo = Item's primary file is under an embargo
- restricted = Item's primary file is not downloadable to anonymous users
- unknown = Item's status is indeterminable or unknown

Reply:
It's a good idea to add a description for the default values. As requested, I added the description you provided with a side note to emphase how the calculation process is done for those values.